src/pip_sync_faster/main.py

```diff
@@ -3,45 +3,30 @@
 import os
 import sys
 from argparse import ArgumentParser
-from functools import lru_cache
+from functools import cached_property
 from importlib.metadata import version
 from pathlib import Path
 from subprocess import CalledProcessError, run
 
 
 class SrcFile:
     def __init__(self, path):
-        self._path = path
+        self.path = path
 
-    @property
-    @lru_cache
-    def path(self):
-        return self._path
```

Comment:
The `path` property was unnecessary, can just use a `path` attribute (as long as no one ever mutates `path` as that would invalidate the cached `abspath` and `contents_hash` properties, but this is really a very small script)